iree/base/dynamic_library.cc

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 Google LLC
```

Comment:
Since there's no real shared code here (that can't be in the dynamic_library.h header) you could make it easier to read by splitting this into dynamic_library_posix.cc and dynamic_library_win32.cc (matching the various file_* things here https://github.com/google/iree/tree/master/iree/base/internal) - you'd still put both in the `srcs` and have the #if defined, but it's easier to find the right file (don't need to scroll around/process #ifdefs, etc). This also tends to make editors happier as they often can't resolve the defines and will try to do intellisense on the wrong platform/etc.

Reply:
Done. I wasn't quite sure how to manage that at first with the subclasses needing to store extra state, but figured it out :)